tests/Microsoft.Identity.Test.Integration.net45/Infrastructure/MsalAssert.cs

```diff
@@ -23,11 +23,15 @@ public static class MsalAssert
             return account;
         }
 
-        public static void AssertAuthResult(AuthenticationResult result, LabUser user)
+        public static void AssertAuthResult(AuthenticationResult result, LabUser user = null)
         {
             Assert.IsNotNull(result);
             Assert.IsNotNull(result.AccessToken);
-            Assert.AreEqual(user.Upn, result.Account.Username);
+
+            if (user != null)
+            {
+                Assert.AreEqual(user.Upn, result.Account.Username);
```

Comment:
>Assert.AreEqual(user.Upn, result.Account.Username); [](start = 16, length = 51)

is this always true? I'm a bit puzzled that upn and Username is the same

Reply:
I am not 100% certain but our tests have not failed because of this and we use a number of different accounts from the lab so I am assuming it is always the same.